R/groupwise_abs_diff.R

```diff
@@ -0,0 +1,24 @@
+#' Groupwise Absolute Differenence (Fairness Metrics)
+#' @name groupwise_abs_diff
+#'
+#' @param prediction The predictions of the learner.
+#' @param base_measure The base measures used to evaluate.
+#' @param data_task The data task for the fairness metric.
+#' @param ... Further arguments, currently ignored.
+#'
+#' @return Absolute difference of base measues between binary protected groups.
+#' @export
+#'
+groupwise_abs_diff <- function(prediction, base_measure, data_task, ...) {
+  #Assert the status for all the parameters
+
+  subcol = data_task$col_roles$pta
+  prediction = split(as.data.table(prediction), data_task$data(cols = subcol))
+  prediction = map(prediction, as_prediction_classif)
+  msr1 = prediction[[1]]$score(base_measure)
+  msr2 = prediction[[2]]$score(base_measure)
```

Comment:
the 5 above lines are repeated in all 3  `groupwise_*` functions. this should probably go into a helper function then.

Reply:
Fixed, create helper function binary_measure_score in helper.R

Reply:
Fixed, create helper function binary_measure_score in helper.R.
However, we may need more helper functions in case of non binary case. But that would some further problems.